### PR TITLE
Implement WINFF.FT.S.SIQ.19

### PR DESCRIPTION
**Pull request by krlinus**
Two spectrum inquiry requests - One request successful and the other
unsuccessful

### CODE REVIEW

**Comment by janetliuboston** [comment]
Please resolve conflicts.

**Comment by krlinus** [comment]
Conflicts are resolved.

File: src/harness/testcases/spectrum_inquiry_testcase.py [hunk 1]
```diff
@@ -206,3 +206,53 @@ def test_WINFF_FT_S_SIQ_15(self):
...
+    }
+    request = {'spectrumInquiryRequest': [
+        spectrum_inquiry_0, spectrum_inquiry_1]}
+    # Send spectrumInquiryRequest and get response
+    response = self._sas.SpectrumInquiry(request)['spectrumInquiryResponse']
+    self.assertEqual(len(response), 2)
+    # Check Spectrum Inquiry Response
+    self.assertEqual(response[0]['cbsdId'], cbsd_ids[0])
+    self.assertTrue('availableChannel' in response[0])
+    for available_channel in response[0]['availableChannel']:
```
Comment (janetliuboston) [inline src/harness/testcases/spectrum_inquiry_testcase.py]: If response[0]['availableChannel'] is null, will this fail?

Add a check?
    if response[0]['availableChannel'] is not None:
        for available_channel in response[0]['availableChannel']:

Comment (krlinus) [inline src/harness/testcases/spectrum_inquiry_testcase.py]: This failure will mean that the test failed. TS says:
"This parameter is an array of zero or more data objects, AvailableChannel, which
describes a channel that is available for the CBSD, see Table 19.
Included: If and only if the Spectrum Inquiry is successful."
It should be an array of objects, even if it contains no elements.

Comment (janetliuboston) [inline src/harness/testcases/spectrum_inquiry_testcase.py]: Agree with this. The test spec needs to be updated to reflect the availableChannel will always be there for successful spectrum inquiry.

**Pull request merged**